### PR TITLE
COM-19943: As an editor, I want to add image captions

### PR DESCRIPTION
**Pull request by clash82**
https://jira.ez.no/browse/COM-19943

screen shot 2016-10-13 at 11 03 44

### CODE REVIEW

**Comment by clash82** [comment]
@SylvainGuittard my suggestion for image caption is to use same styles as we used for code snippets. Let me know what do you think?

**Comment by SylvainGuittard** [comment]
I think the border used for the code is too dark. I need to check with Hakan.

**Comment by clash82** [comment]
ok, waiting for more details

**Comment by SylvainGuittard** [comment]
We have a new design! See https://jira.ez.no/browse/COM-19958

File: web/assets/css/embed.css [hunk 1]
```diff
@@ -16,3 +16,35 @@
...
+.embed-code {
+    background: #4c4c4c;
+    padding: 5px;
+    margin: 0;
+}
+
+.embed-code pre {
+    border-radius: 0;
+    margin-bottom: 5px;
+    background: #f2f2f2 ;
```
Comment (SylvainGuittard) [inline web/assets/css/embed.css]: CS: spacing after ```#f2f2f2```

Comment (clash82) [inline web/assets/css/embed.css]: Fixed, thank you.

**Comment by clash82** [comment]
review please @damianz5

**Pull request merged**